zipf/bin/plotcounts.py

```diff
@@ -109,5 +117,9 @@ def main(args):
                         help='YAML file containing matplotlib parameters')
     parser.add_argument('--style', type=str, nargs='*', choices=plt.style.available,
                         default=None, help='matplotlib style')
+    parser.add_argument('--saveconfig', type=str, default=None,
+                        help='Save configuration to file')
+    parser.add_argument('--loadconfig', type=str, default=None,
```

Comment:
We don't need the `--loadconfig` option because the existing `--plotparams` option already takes a YAML file full of matplotlib parameters.